packages/accounts/src/msca/multi-owner-account.ts

```diff
@@ -57,11 +62,12 @@ export const createMultiOwnerMSCABuilder = <
     .withExecutor(StandardExecutor)
     .withSigner((acct) => {
       const signWith1271Wrapper = async (msg: Hash): Promise<`0x${string}`> => {
-        const multiOwnerAugmented =
-          acct.extendWithPluginMethods(MultiOwnerPlugin);
+        const multiOwnerTokenReceiverAugmented = acct
+          .extendWithPluginMethods(TokenReceiverPlugin)
```

Comment:
shouldn't this respect the flag to exclude it?

Reply:
actually I don't even need this to be augmented with token receiver plugin methods here because not needed here.